src/test/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintResponseMessageListenerIntegrationTest.kt

```diff
@@ -115,7 +115,9 @@ internal class ProcessPrintResponseMessageListenerIntegrationTest : IntegrationT
         val expectedEmailMessage = buildLocalstackEmailMessage(
             timestamp = OffsetDateTime.now(UTC).toLocalDateTime().truncatedTo(SECONDS),
             emailSender = "noreply_erouser@erop.ierds.uk",
-            toAddresses = setOf("a-user@valtech.com"),
+            // TODO - remove this as soon as EIP1-4998 and EIP1-5090 have been tested
+            toAddresses = setOf("joshua.ayoade@valtech.com"),
```

Comment:
I suggest we also temporarily change log.debug to log.info or add few log info before and after email is actually sent